Added BreakLines package details

Moojuiceman opened the pull request:
<!--
Your pull request will be reviewed automatically and by a human.

The manual review may take several days or weeks, depending on the reviewer's availability and workload.
If you haven't received a comment on your pull request and it wasn't merged either,
it just hasn't been reviewed yet.

---

Please ensure the automated reviews pass. 
Follow the instructions provided, if necessary.
You can speed up the process
by [running some tests locally](https://packagecontrol.io/docs/submitting_a_package#Step_7).

You can trigger @packagecontrol-bot to re-evaluate your pull request
by pushing a commit or closing and reopening your pull request.
Do **NOT** open a new pull request!

In general, make sure you:

 1. Used `"tags": true` and not `"branch": "master"` 
    (versioning docs: <https://packagecontrol.io/docs/submitting_a_package#Step_4>)
 2. Added a README to your repository so that users (and reviewers) 
    can understand what your package provides.
 
You should proceed with a short description of what the package does
and, in case one or multiple similar package already exists, 
why you believe it is different and needed
below this line. -->

Thom1729 commented:
- You should add a license.
- It would be better to use a `TextCommand` for `PromptBreakLinesCommand` so that you can use `self.view` instead of `self.window.active_view()`. You can access the current window via `self.view.window()`.
- In `BreakLinesCommand`, is it necessary to handle invalid `break_gap`? Nothing should be passing in invalid values, and if something does, it's probably better to let the command fail and an error appear in the console.
- In `break_lines`, you can probably simplify the logic by iterating over `range(first_line, last_line + 1, break_gap)[1:]` in reverse.

Moojuiceman commented:
@Thom1729 
- Added MIT License
- Updated as recommended
- The menu/hotkey commands were passing the variable as a string. These have been changed and try/catch removed
- Updated as recommended, but with first_line + break_gap otherwise an extra break is added before the first line

If you have any other recommendations let me know, otherwise I'll re-tag so the changes are included in the package control build
